codec/src/test/resources/io/netty/handler/codec/xml/sample-04.xml

```diff
@@ -457,14 +457,14 @@
       </plugin>
       <plugin>
         <artifactId>maven-deploy-plugin</artifactId>
-        <version>2.7</version>
+        <version>2.8.2</version>
         <configuration>
           <retryFailedDeploymentCount>10</retryFailedDeploymentCount>
         </configuration>
       </plugin>
       <plugin>
         <artifactId>maven-release-plugin</artifactId>
-        <version>2.4.1</version>
+        <version>2.5.3</version>
```

Comment:
So, it's probably a good idea to add some comment? e.g. Revert to 2.4.1 if release process fails due to git-related errors.